modulefiles/hera.lua

```diff
@@ -0,0 +1,70 @@
+help([[
+loads hafs_tracker modulefile and related set environment veriables on Hera
```

Comment:
Double check comments in modulefiles to make sure they are accurate for the package name and machine.